### ISSUE TITLE
Text alignment or left/right padding?

### ISSUE THREAD

**Issue by jsymon** [tktonjy/customizable_editor_tabs#3]
Very nice plugin. 
Are there any plans to add text alignment (left / right / center) or to specify a fixed number of pixel space to the left/right of the text label?

**Comment by tktonjy** [comment]
Yeah I can do that. By the way I'm doing this plugin in my spare time. And I've received so far 5 dollars in donations from 3000 downloads :)
